fabric-networking-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/event/ServerboundPacketChannelCallback.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.networking.v1.event;
+
+import net.minecraft.server.network.ServerPlayNetworkHandler;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+/**
+ * Holds event instances of client to server custom payload packet channel registration.
+ */
+public interface ServerboundPacketChannelCallback extends PacketChannelCallback<ServerPlayNetworkHandler> {
```

Comment:
I think `C2S` (like in the packet class names) would be better.

Reply:
Should just call it Server and Client as those are where the events happen.